mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_cluster_test.go

```diff
@@ -70,27 +70,7 @@ func TestAccVmwareengineCluster_vmwareEngineClusterUpdate(t *testing.T) {
 func testVmwareEngineClusterConfig(context map[string]interface{}, nodeCount int) string {
 	context["node_count"] = nodeCount
 	return acctest.Nprintf(`
-resource "google_project" "project" {
```

Comment:
Would you mind clarifying the change here a bit? Are test not needed to be wrapped in its own project any more? Are you happen to know why it's needed during the initial implementation? Thanks

Reply:
Vmwareengine tests require node provisioning due to the creation of Private Clouds and Clusters. 

Last month, tests were wrapped in custom created projects in #9835 due to some quota failures. However, this resulted in multiple errors which were eventually filed in b/326669990. This is because the newly created test projects di not have enough node quota to run these tests. 

The nightly test project has a quota of 21 nodes assigned to it, and has the capability to run all tests parallely without running into any quota issues. The reason why these quota issues were arising in the first place was because of dangling Private Clouds not being cleaned up (Which is to be solved by adding handwritten sweepers here).